README.rst

```diff
@@ -233,7 +233,13 @@ The contents of this key file should be a list of age X25519 identities, one
 per line. Lines beginning with ``#`` are considered comments and ignored. Each
 identity will be tried in sequence until one is able to decrypt the data.
 
-Encrypting with SSH keys via age is not yet supported by SOPS.
+Encrypting with SSH keys via age is also supported by SOPS. You can use SSH public keys
+("ssh-ed25519 AAAA...", "ssh-rsa AAAA...") as age recipients when encrypting a file.
+When decrypting a file, SOPS will look for `~/.ssh/id_ed25519` and fallbacks to
+`~/.ssh/id_rsa`. You can specify the location of the private key manually by setting
```

Comment:
Since this is RST, you need double backticks:
```suggestion
When decrypting a file, SOPS will look for ``~/.ssh/id_ed25519`` and falls back to
``~/.ssh/id_rsa``. You can specify the location of the private key manually by setting
```